test/snapshot/function.test.js

```diff
@@ -91,4 +91,21 @@ describe("Function tests", function () {
     });
     expect(astErr).toMatchSnapshot();
   });
+
+  it("test danging comma in function php 8.0", function () {
+    const astErr = parser.parseEval(
+      `function (
+    $a,
+    $b,
+    $c,
+  ) {}`,
+      {
+        parser: {
+          version: "8.0",
+          suppressErrors: true,
```

Comment:
Do we need to suppress errors here?

Reply:
No we don't, I have removed that option.